setup/Directory.Build.targets

```diff
@@ -22,6 +22,11 @@
     </ItemGroup>
   </Target>
 
+  <!-- Updates insertion manifests (json) for creating the vsman files for insertion to include the SPDX SBOM json. -->
+  <Target Name="UpdateManifestJsonForSbom" AfterTargets="AddVsixForSigning">
+    <Exec ContinueOnError="false" Command="powershell -NonInteractive -NoLogo -NoProfile -ExecutionPolicy Unrestricted -Command &quot;. $(RepoRoot)build\script\UpdateManifestJsonForSbom.ps1 -manifestJsonPath '$(VisualStudioSetupInsertionPath)$(VisualStudioInsertionComponent).json' -sbomMetadataPath '$(ArtifactsBinDir)spdx_2.2\manifest.spdx.json'&quot;" />
+  </Target>
```

Comment:
Technically, this should be `AfterTargets="$(AddVsixForSigningDependsOn)"` as that value (target) indicates when VSIX packaging has completed. Doing it as `AfterTargets="AddVsixForSigning"` unnecessarily binds the SBOM process to signing. But there is not currently a scenario where we wouldn't sign and do SBOM.

Reply:
Your description makes sense. It'd be good as a comment in the file.